akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -471,7 +471,8 @@ private[remote] abstract class ArteryTransport(_system: ExtendedActorSystem, _pr
         bindAddress.uid)
     else {
       log.info(
-        s"Remoting started with transport [Artery ${settings.Transport}]; listening on address [{}] and bound to [{}] with UID [{}]",
+        "Remoting started with transport [Artery {}]; listening on address [{}] and bound to [{}] with UID [{}]",
+        settings.Transport,
```

Comment:
I wonder if that wasn't on purpose to avoid triggering the vararg/array allocation

Reply:
@patriknw knows

Reply:
I'm not sure I understand - in https://github.com/akka/akka/blob/master/akka-actor/src/main/scala/akka/event/Logging.scala#L1361 it looks like a vararg is created for either 3 or 4 parameters?

Reply:
Ah, I thought we had a 3 param overload but not 4, I was wrong then! :)

Reply:
We don't even have vargars for the log methods (`info`, `debug`, etc).